src/ncp-spinel/SpinelNCPControlInterface.cpp

```diff
@@ -328,6 +329,39 @@ SpinelNCPControlInterface::joiner_add(
 	return;
 }
 
+void
+SpinelNCPControlInterface::joiner_attach(CallbackWithStatus cb)
+{
+	mNCPInstance->start_new_task(SpinelNCPTaskSendCommand::Factory(mNCPInstance)
+			.set_callback(cb)
+			.add_command(SpinelPackData(
+					SPINEL_FRAME_PACK_CMD_PROP_VALUE_SET(SPINEL_DATATYPE_BOOL_S),
+					SPINEL_PROP_NET_STACK_UP,
+					true
+					))
+			.finish()
+			);
```

Comment:
`joiner_attach` is setting `NET_STACK_UP` property to `true`?

Can we use the existing `SpinelNCPControlInterface::attach()` instead?

Reply:
Existing `attach()` has both `NET_IF_UP` and `NET_STACK_UP`, however the interface has been up during previous commissioning process. No atomic `NET_IF_UP` and  `NET_STACK_UP` in existing implementation, that's why here introduces the `joiner_attach()` to avoid redundant `NET_IF_UP` operation.
I don't have strong thoughts on this, could reuse `attach()` if you think it's fine with the redundant `NET_IF_UP` operation.

Reply:
Thanks. Let's keep the `joiner_attach`.